_posts/2016-03-21-an-event-apart-nashville.md

```diff
@@ -0,0 +1,160 @@
+have to use the internet.
+
+Josh Clark brought this up as well when discussing the rapid development of new
+inputs for touch devices, recognizing that we don't yet have a set of principles
+to design by. His simple call to action:
+
+> Be the person who cares.
+
+This principle was demonstrated throughout the conference, too. Even when
+presenters brought up examples of people getting things wrong, they did so to
+ensure that we learn from each other's mistakes rather than repeating them, and
+they constantly reiterated that we all have room to improve.
+
+### Other awesome things
+
+Here's a little data-dump of other noteworthy items:
+
+- From Jeffrey Zeldman: **We’re facing the same problems we always have** - bloat,
+arguments over what code/language to use/support, which browsers to support,
+different screen sizes/dimensions, how to do layouts, etc. These aren't new
+problems even though the web is constantly evolving.
+- From Jen Simmons: **The
+[feature query](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports).**
+Incomplete CSS3 modules are no longer unusable; they can be used in browsers
+that support them and ignored by browsers that don't.
+- Speaking of incomplete CSS3 modules,
+**[CSS shapes](http://alistapart.com/article/css-shapes-101).** This one's a ways
+off from being completed, but I can't wait to play around with it.
+- From Aaron Gustafson: **Work harder so your users don't have to.** Aaron's
+comprehensive guide on forms emphasized carefully considering every form field
+to place the brunt of the work on the developer rather than the user.
+- From Josh Clark: **Don't require a different app for every smart device.**
+Make it easy on the user to connect via
+[the physical web](https://google.github.io/physical-web/).
+- From Jason Grigsby: **Input is exploding.** Consider the following: keyboard,
+mouse, camera, accelerometer, trackpad, gestures, location services, voice,
+compass, beacons (e.g. bluetooth), gyroscope, Siri and other AIs, fingerprint
+recognition, Apple Pay, barometer, 3D touch. There were 142 years between the
+keyboard and the mouse, and now we get new inputs with each new device.
+- From Josh Clark: when designing for touch, **let the real world be your guide.**
+Function isn't always more important than form when it comes to enabling new
+users to seamlessly learn new technologies. As data becomes more and more
+tangible, we can capitalize on new inputs to allow users to interact with the
+web in increasingly realistic, physical ways.
+
+### Was it worth it?
+
+This was my biggest question given the relatively high cost, and yes, I think it
+was worth it. The high caliber of the speakers, their presentations, and the
+event as a whole justified the ticket price in my opinion. Every talk was
+polished, informative, and relevant to the current status and future of our
+industry. Not only did a learn a ton, I believe that the event spreads a
```

Comment:
Need to reword this sentence, I think there's a word missing...

Not only did "I" learn a ton,.....